LibertyDSNP/example-client
# Show DSNP icons

Purpose
---------------
This completes the work on showing avatars. We need to display the icons that come from user's profiles rather than generating icons.

Solution
---------------
This PR pulls the first icon from the profile and displays it everywhere we show UserAvatars (Profile block, posts, new post dialog, reply's, etc.). It also modifies the populate script to store avatars on the server and associate one with each profile.

Change summary
---------------
* Add open licensed avatars for each account in the populate script.
* Modify populate script to store avatar images on web server and add a link to each profile.
* Modify `UserAvatar` to take an optional `icon` property.
* Fetch url from profile and use it to display icon in (almost) all places where we need to show avatars. 

Note: The connections lists is currently still completely mocked out using old assumptions about how we retrieve data. Showing avatars the right way proved to be more trouble than it's worth since that feature isn't really working until we get it using DNSP data, so it's still showing generated avatars for now.

Steps to Verify
----------------
1. Restart the app with a clean, deployed chain.
2. Run the populate script.
3. Note that every place you can see a user avatar, it's now an image of a human face and that the faces are consistent with names and handles across the app.
